.expeditor/verify.pipeline.yml

```diff
@@ -73,6 +73,23 @@ steps:
             "/go/src/github.com/chef/automate"
           ]
 
+  - label: ":semgrep: Published"
+    expeditor:
+      executor:
+        docker:
+          image: returntocorp/semgrep:latest
+          entrypoint: semgrep
+          command: [
+            "--error",
+            "--exclude", "third_party",
+            "--exclude", "*_test.go",
+            "--exclude", "*.pb.go",
+            "--exclude", "*.spec.ts",
+            "--timeout", "120",
+            "--config", "https://semgrep.dev/p/r2c-ci",
```

Comment:
This is the published ruleset I am using as a "starter". We can add more when we want.